static/preference.md

```diff
@@ -9,6 +9,7 @@ Edit and save to update preferences.  You can only change the JSON below!
 ```json
 {
   "fontSize": "16px",
+  "editorFontFamily": "Monaco",
```

Comment:
Can you please change default font family to `Open Sans`.

Reply:
Done!